scripts/make-subpkg.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Still trying to understand the workflow. How do you currently upload the tar file?